resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -0,0 +1,415 @@
+---
+
+- slug: trimmomatic-single
+  name: trimmomatic (single-end)
+  data_name: '{{ reads.fastq.0.file|default:"?" }}'
+  version: 0.0.1
+  type: data:reads:fastq:single:trimmomatic
+  category: analyses
+  flow_collection: sample
```

Comment:
This line is not required and will also be removed from other reads-filtering processors.